libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -88,9 +109,14 @@ export class FixedCardLayoutComponent implements OnInit, AfterContentInit, After
     /** @hidden */
     private _previousNumberOfColumns: number;
 
+    private _cardsArray: Array<CardDefinitionDirective>;
```

Comment:
Let's add `/** @hidden */` comment, to hide it from docs

Reply:
added /** @hidden */ to hide from API docs.